app/models/user.rb

```diff
@@ -5,4 +5,6 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_one :profile, dependent: :destroy
   has_many :posts, dependent: :destroy
+
+  enum role: { user: 0, admin: 2 }
```

Comment:
Em alguns contextos, os times de desenvolvimento preferem usar valores mais espaçados, por exemplo, user 0 e admin 10, pra deixar bastante espaço entre os dois. Assim, se for necessário no futuro, tem bastante espaço para incluir outros papéis entre user e admin. 
O que vocês fizeram não está incorreto, isso é só um comentário sobre enum.